test/e2e/config/ironic.yaml

```diff
@@ -36,3 +40,6 @@ intervals:
   default/wait-namespace-deleted: ["10m", "1s"]
   ironic/wait-deployment: ["10m", "2s"]
   default/wait-power-state: ["10m", "100ms"]
+  default/wait-provisioning: ["20s", "1s"]
+  default/wait-provisioned: ["5m", "5s"]
```

Comment:
Wow this is super fast compared to the provisioning in CAPM3! I guess that tiny image helps quite a lot :smile: